include/xsparse/levels/compressed.hpp

```diff
@@ -58,6 +58,12 @@ namespace xsparse
                                                                   LevelProperties>;
 
         public:
+            // Function to access the LevelProperties object
```

Comment:
`constexpr` doesn't have to be compile-time, a `using` clause would be better for all our compile-time needs.

Reply:
If we do using, how would we access the property dynamically?

Reply:
@bharath2438 Perhaps you can answer this better, at least until I have my PC. 😅

Reply:
But you can use `if constexpr` or `if consteval` and `static_assert`.

Reply:
Sorry I don't think I follow 100% 😐

Do you mean that we use the if constexpr inside the coiteration code to check the namespace levels? How would we do that? I didn't see a way of doing so.

Reply:
Wouldn't renaming the template parameter to `_LevelProperties` then doing `using LevelProperties = LevelProperties;` in a `public` block; then `decltype(level)::LevelProperties::property_name` work?

Does that answer the question better?

Reply:
Ah yes that makes sense. I wasn't even thinking of this approach.